README.md

```
@@ -1,93 +1,64 @@
+Library:   
+- [Retrofit2 & OkHttp3](https://github.com/square/retrofit)   
+- [Coil](https://coil-kt.github.io/coil/getting_started/)   
+
+
+## 🏛️ Architecture(Modularization) Overview   
+<p align="start"><strong>We follow the Google App Architecture Guidelines.</strong></p>
+
+<table align="center">
+  <tr>
+    <td width="70%">
+      <img src="https://github.com/user-attachments/assets/a16ca52d-3846-42c6-acef-b195c947fb34" width="100%" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)